CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ set (CMAKE_USE_RELATIVE_PATHS 1)
 
 OPTION (ENABLE_DOCS "Build the documentation" OFF)
 OPTION (ENABLE_GTESTS "Enable building and running GTest unit tests" ON)
+OPTION (ENABLE_TOFINO "Build the TOFINO backend (required for the full test suite)" ON)
```

Comment:
I would disable this by default and only build Tofino on the Ubuntu 22.04 Github actions.

Reply:
> Another thing is that this backend is really... large :) So not all breaking changes might be quite non-trivial :)

This might actually be a good thing. bf-p4c pushes the limits of what you can do as a back end. So if we introduce breaking changes it can act as a canary.

Reply:
Yes, it has both pros and cons. I am talking mostly about the amount of possible changes required if we'd introduce, say, some IR change or something like this :)